modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -44,6 +46,23 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
     });
   });
 
+  const setSizingMode = (sizing: string) => TableSelection.getSelectionStartCellOrCaption(editor).each((cellOrCaption) => {
```

Comment:
```suggestion
  const setSizingMode = (sizing: 'relative' | 'fixed' | 'responsive' | ...) => TableSelection.getSelectionStartCellOrCaption(editor).each((cellOrCaption) => {
```

Reply:
We've had problems elsewhere with these kinds of types so I tend to avoid them. Would be nice though...

Reply:
It's coming from user input, so this would give a false sense of security, hence why it's `string`.